libs/light/docs/reference/light.md

```diff
@@ -40,8 +40,8 @@ light.createStrip().setPhotonMode(PhotonMode.PenUp)
 light.rgb(255, 255, 255);
 light.hsv(255, 255, 255);
 light.colors(Colors.Red);
-//light.fade(0, 0)
-//light.animation(LightAnimation.Rainbow)
+light.fade(0, 0)
+light.animation(0)
```

Comment:
this one is legacy, let's remove it from the docs

Reply:
Ok. 

Also, are we good with the layout having the strip related APIs under _/light/neopixelstrip/*_?